File: corpus/native/obfuscators/rust-obfuscator/sample.rs
fn classify(n: i32) -> i32 {
    if n > 10 {
        n * 2
    } else {
        n + 1
    }
}

fn main() {
    let secret = "the-hidden-flag-value";
    println!("classify={},{}", classify(7), classify(20));
    println!("secret={}", secret);
}

File: corpus/native/obfuscators/obfush/sample.c
#include <stdio.h>

int classify(int n) {
    if (n > 10) {
        return n * 2;
    } else {
        return n + 1;
    }
}

int main(void) {
    printf("classify=%d,%d\n", classify(7), classify(20));
    printf("secret=%s\n", "the-hidden-flag-value");
    return 0;
}
